src/cf-notification/src/cf-notification.less

```diff
@@ -0,0 +1,44 @@
+/* ==========================================================================
+   Capital Framework
+   Notifications Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// Color variables
```

Comment:
I didn't [include the 18F reference](https://github.com/cfpb/capital-framework/blob/canary/src/cf-pagination/src/cf-pagination.less#L12) (which is also a dead link and needs an update) as I wasn't sure what we're doing with this going forward? How should I handled this?

Reply:
Done!